### PR TITLE
[docs][sc-105732] Update the Fluent Bit monitoring page

### PR DESCRIPTION
**Pull request by lockewritesdocs**
This PR updates the [Fluent Bit monitoring page](https://docs.fluentbit.io/manual/administration/monitoring) for style and consistency with Chronosphere guidelines.

### CODE REVIEW

File: administration/monitoring.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
 <img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=e9ca51eb-7faf-491d-a62e-618a21c94506" />
 
-# Monitoring
+# Monitor data pipelines
 
-Fluent Bit comes with built-it features to allow you to monitor the internals of your pipeline, connect to Prometheus and Grafana, Health checks  and also connectors to use external services for such purposes:
+Fluent Bit includes features for monitoring the internals of your pipeline, and for
+connecting to Prometheus and Grafana, Health checks, and connectors to use external
+services:
```
Comment (esmerel) [inline administration/monitoring.md:12]:
```suggestion
Fluent Bit includes features for monitoring the internals of your pipeline, 
connecting to Prometheus and Grafana, Health checks, and connectors to use external
services:
```

multiple ands in there feels awkward

File: administration/monitoring.md [hunk 2]
```diff
@@ -80,132 +93,150 @@ $ curl -s http://127.0.0.1:2020 | jq
...
+| /api/v1/metrics            | Display internal metrics per loaded plugin. | JSON                  |
+| /api/v1/metrics/prometheus | Display internal metrics per loaded plugin in Prometheus Server format. | Prometheus Text 0.0.4 |
+| /api/v1/storage            | Get internal metrics of the storage layer / buffered data. This option is enabled only if in the `SERVICE` section of the property `storage.metrics` is enabled. | JSON |
+| /api/v1/health             | Display the Fluent Bit health check result. | String |
+| /api/v2/metrics            | Display internal metrics per loaded plugin. | [cmetrics text format](https://github.com/fluent/cmetrics) |
+| /api/v2/metrics/prometheus | Display internal metrics per loaded plugin ready in Prometheus Server format. | Prometheus Text 0.0.4 |
+| /api/v2/reload             | Execute hot reloading or get the status of hot reloading. See the [hot-reloading documentation](hot-reload.md). | JSON |
+
+### v1 metrics
+
```
Comment (esmerel) [inline administration/monitoring.md:113]: should be an sentence here between headers but it's not a blocker, same for below, but that can totally be iterated later.

**Review by esmerel — APPROVED** [review]
minor comment but this is MUCH cleaner.

File: administration/monitoring.md (hunk 1 above)
Comment (lockewritesdocs) [reply administration/monitoring.md:12]: Mmm, yeah -- good callout! I'll change to read:

>Fluent Bit includes features for monitoring the internals of your pipeline, in
>addition to connecting to Prometheus and Grafana, Health checks, and connectors to
>use external services:

File: administration/monitoring.md (hunk 2 above)
Comment (lockewritesdocs) [reply administration/monitoring.md:113]: ACK -- will add a sentence there 👍

**Pull request merged**